3rdparty/pine/pine_server.h

```diff
@@ -27,7 +27,7 @@
 #include <sys/un.h>
 #include <arpa/inet.h>
 #include <poll.h>
-#if defined(__DragonFly__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__APPLE__)
+#if defined(__DragonFly__) || defined(__FreeBSD__) || defined(__linux__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__APPLE__)
```

Comment:
You could just do `#ifndef MSVC` at this point

Reply:
sure

Reply:
Wait, it's right above this. Why was this even separated lol.

Reply:
Since it's a platform check, _WIN32 would be the right define to check for.